app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/App.java

```diff
@@ -13,6 +13,7 @@ public class App {
     public static void main(String[] args) {
         var app = Javalin.create().start(8080);
 
+        app.get("/", ctx -> ctx.result("Welcome to the CDC TI service"));
```

Comment:
Should this get removed?  I don't personally see a reason for this, but I'm curious what others think?

Reply:
I'm ok with removing it.

Reply:
I'm cool with removing it. However, without something to respond to a 'GET /' against the app's Azure deployment url, the result is a 404. In the future I expect we may want to direct the guest to our documentation or other information about how to use the API.